### ISSUE TITLE
Provide support for Obsidian 1.4 new Frontmatter

### ISSUE THREAD

**Issue by thibaultmol** [joethei/obsidian-link-favicon#64]
It would be nice if you put a link as property's value in the new 1.4 Properties feature, it would show the favicon like it would in the regular note text

**Comment by joethei** [comment]
Good idea, will wait a bit for properties to stabilize/an official API to be released, before I implement this.

**Comment by thibaultmol** [comment]
Fair enough. Thx again for making this plugin. It's a small thing, but it makes Obsidian a lot more visually pleasant to look at

**Comment by NomarCub** [comment]
The official API is stable now, Obsidian 1.4 is out.

**Comment by joethei** [comment]
It's not stable yet, there are still things that have not been implemented.
And the properties API is not released yet.

(Disclaimer: I am part of the Obsidian team)

**Comment by Kellenok** [comment]
any news?
